### PR TITLE
Upgrade prometheus

### PR DESCRIPTION
**Pull request by res-life**
Upgrade promethues to v2.37.6

```
git clone git@github.com:prometheus/prometheus.git
git checkout v2.37.6
cat /home/chongg/code/prometheus/go.mod | grep "golang.org/x/"
golang.org/x/net v0.7.0
```

### CODE REVIEW

**Comment by res-life** [comment]
Still a draft, need to verify the new image.

**Comment by res-life** [comment]
I've verified the functionality successfully.

**Comment by res-life** [comment]
@GaryShen2008  help review

File: Databricks/Dockerfile [hunk 1]
```diff
@@ -255,7 +255,7 @@ RUN cp ${ALLUXIO_HOME}/client/alluxio-${ALLUXIO_VERSION}-client.jar /databricks/
 #############
 # Setup Prometheus
 #############
-ARG PROMETHEUS_VERSION=2.37.3
+ARG PROMETHEUS_VERSION=2.37.6
 ARG PROMETHEUS_TAR_FILE=prometheus-${PROMETHEUS_VERSION}.linux-amd64.tar.gz
 ARG PROMETHEUS_DOWNLOAD_URL=https://github.com/prometheus/prometheus/releases/download/v${PROMETHEUS_VERSION}/${PROMETHEUS_TAR_FILE}
```
Comment (tgravescs) [inline Databricks/Dockerfile:260]: update copyright

File: Databricks/init.sh [hunk 2]
```diff
@@ -39,7 +39,7 @@ ALLUXIO_WORKER_DIRECT_HEAP=${ALLUXIO_WORKER_DIRECT_HEAP:-'4g'}
 ALLUXIO_COPY_LOG_PATH=${ALLUXIO_COPY_LOG_PATH:-''}
```
Comment (tgravescs) [inline Databricks/init.sh:39]: update copyright

File: Databricks/Dockerfile (hunk 1 above)
Comment (res-life) [reply Databricks/Dockerfile:260]: Done

File: Databricks/init.sh (hunk 2 above)
Comment (res-life) [reply Databricks/init.sh:39]: Done

**Review by tgravescs — APPROVED** [review]

**Pull request merged**